content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+
+Those tabs are: :ref:`Tickets <create_events/event_form/tickets>`, :ref:`Communication
+<create_events/event_form/communication>`, :ref:`Questions <create_events/event_form/questions>`,
+and :ref:`Notes <create_events/event_form/notes>`.
+
+.. _create_events/event_form/tickets:
+
+Tickets tab
+-----------
+
+Create custom tickets (and ticket tiers) for events in the :guilabel:`Tickets` tab of an event form.
+
+.. image:: create_events/tickets-tab.png
+   :align: center
+   :alt: A typical tickets tab on an event form in the Odoo Events application.
+
+To create a ticket, click :guilabel:`Add a line` in the :guilabel:`Tickets` tab. Then, enter a name
+for the ticket (e.g. `Basic Ticket` or `VIP`) in the :guilabel:`Name` field.
+
+Next, in the :guilabel:`Product` field, either select a pre-configured event registration product,
+which Odoo creates when the *Events* app is installed, or create a new one directly from this field.
+
+.. important::
+   Upon installing Odoo *Events* a new product type, *Event Ticket*, becomes selectable on product
+   forms. In order for an event registration product to be selectable in the :guilabel:`Tickets` tab
+   of an event form, the event registration product type **must** be set to *Event Ticket*.
+
+.. note::
+   To create a new event registration product directly from this field, type in the name of the new
+   event registration product, and select either :guilabel:`Create` or
+   :guilabel:`Create and edit...` from the resulting drop-down.
+
+   Clicking :guilabel:`Create` creates the event registration product, which can be edited later.
+   Clicking :guilabel:`Create and edit...` creates the event registration product, and allows the
+   user to edit its configurations right away on a :guilabel:`Create Product` pop-up window.
+
+.. tip::
+   Existing event registration products can be modified directly from this field, as well, by
+   clicking the :guilabel:`➡️ (right arrow)` icon, located beside the event registration product.
+   Doing so reveals that product's form. Then, proceed to make any desired modifications.
+
+.. note::
+   In a database with the *Inventory* application installed, clicking the :guilabel:` ➡️ (right
+   arrow)` icon, located beside the event registration product reveals the product form inside the
+   *Inventory* application, with additional choices available to customize.
+
+   However, in a database *without* the *Inventory* application installed, clicking the
+   :guilabel:`➡️ (right arrow)` icon, located beside the event registration product reveals the
+   product form, but only with minimal fields and customizations visible.
+
+.. note::
+   Installing *just* the *Events* application on a trial database installs additional applications
+   automatically: :guilabel:`Discuss`, :guilabel:`Sales`, :guilabel:`Dashboards`,
+   :guilabel:`Invoicing`, and :guilabel:`Website`.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field.
```

Comment:
I think it is an odd place to state here in the Ticket instructions the applications that are installed as dependancies. I'd suggest placing this admonition either at the top of the doc or maybe in the event essentials doc?

Also, to help specify that the Price field sets the registration price for an attendee– modify the sentence 
```suggestion
Next, set the registration cost of the ticket in the :guilabel:`Price` field.
```